cloud/shared/bin/run.py

```diff
@@ -0,0 +1,63 @@
+#! /usr/bin/env python3
+
+import argparse
+import os
+import sys
+import importlib
+
+sys.path.append(os.getcwd())
+
+from cloud.shared.bin.lib.config_loader import ConfigLoader
+from cloud.shared.bin.lib.write_tfvars import TfVarWriter
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '--tag', help='Civiform image tag. Required for Setup and Deploy.')
+    parser.add_argument(
+        '--command',
+        help='Command to run. If ommited will validate config and exit.')
+    parser.add_argument(
+        '--config',
+        default='civiform_config.sh',
+        help='Path to civiform config file.')
```

Comment:
nit: to disambiguate with `application.conf` maybe say "Path to CiviForm deployment config file"